test/micro/org/openjdk/bench/java/lang/foreign/AllocFromTest.java

```diff
@@ -164,6 +164,7 @@ public void close() {
         }
 
         @Override
+        @SuppressWarnings("removal")
```

Comment:
I believe we already disable a bunch of warnings from the command line when compiling these benchmarks. Perhaps we can just tweak the build script in a similar way and avoid the changes to the sources? E.g.

```
make/test/BuildTestLib.gmk:    DISABLED_WARNINGS := try deprecation rawtypes unchecked serial cast removal preview restricted dangling-doc-comments, \
```

Reply:
> I believe we already disable a bunch of warnings from the command line when compiling these benchmarks. Perhaps we can just tweak the build script in a similar way and avoid the changes to the sources? E.g.
> 
> ```
> DISABLED_WARNINGS := restricted this-escape processing rawtypes cast \
> ```
> 
> Should we add `removal` there?

That might be better, I assumed the micros were being compiled an empty DISABLED_WARNINGS list. It means that the FFM micros won't need to be updated.